packages/ra-ui-materialui/src/field/ReferenceField.tsx

```diff
@@ -255,5 +255,9 @@ const Root = styled('span', {
     name: PREFIX,
     overridesResolver: (props, styles) => styles.root,
 })(({ theme }) => ({
-    [`& .${ReferenceFieldClasses.link}`]: {},
+    [`& .${ReferenceFieldClasses.link}`]: {
+        '& :first-child': {
```

Comment:
I'd use the child selector (`> *`) instead to select all children if there are more than one.